### ISSUE TITLE
Move to SUIT org

### ISSUE THREAD

**Issue by simonsmith** [giuseppeg/suitcss-toolkit#6]
Have you considered this?

The more generic ones like `form`, `module` and `card` could be very useful alongside the other core components as they aren't overly opinionated and I'm sure projects ends up recreating something similar.

Wasn't sure if that would disrupt your lerna setup though.

Otherwise including a link in the `suit` README will be a nice addition.

**Comment by giuseppeg** [comment]
Yep I just used Lerna because it makes it easy to manage the packages.
As I started to use them on a real project I found some bugs and things that I would like to fix

For example:
- now I agree that `Form` should be `FormField` - I might rewrite it and release v2 or make a separate component
- `Module` it is a bit overkill I'd rather use spacing utilites (I will open source them). 
- Haven't used `Card` but it should work fine.

**Comment by simonsmith** [comment]
Interesting, okay that makes sense. 

I suppose Module is in the same bracket as spacing utils. Maybe it's too tricky to have a one size fits all component for spacing. I remember the spacing classes got deprecated for a similar reason.

I sometimes use this to make it less hassle - https://github.com/simonsmith/postcss-generate-preset

**Comment by giuseppeg** [comment]
Module requires a lot of boilerplate, you need a UL-like structure even when you just want to add space between two elements.

I am using a scale `n=1...6` of spacing utils with the following formula 

``` css
.u-margin0 { margin: 0 !important }
.u-marginN { margin: calc(var(--u-margin) * 2^(n-1))  !important }
```

and it works great.

**Comment by giuseppeg** [comment]
... not to mention that have responsive companion which makes it super easy to tweak things at different screen sizes.

**Comment by giuseppeg** [comment]
it seems that this guy read my mind and made a framework and a talk https://twitter.com/giuseppegurgone/status/778843098852036608 :D
I am "tired" of writing css when I am asked to build features.
